packages/rs-dpp/src/voting/contender_structs/mod.rs

```diff
@@ -141,3 +146,43 @@ pub struct Contender {
     /// The vote tally for the contender.
     pub vote_tally: Option<u32>,
 }
+
+impl From<FinalizedContender> for Contender {
+    fn from(value: FinalizedContender) -> Self {
+        let FinalizedContender {
+            identity_id,
+            document,
+            final_vote_tally,
+            ..
+        } = value;
+
+        Contender {
+            identity_id,
+            document: Some(document),
+            vote_tally: Some(final_vote_tally),
+        }
+    }
+}
```

Comment:
I am not sure why the SDK would want this.

Reply:
We return Contender in Sdk, not FinalizedContender. Do you mean this is incorrect?

Reply:
Removed `impl From<FinalizedContender> for Contender`